packages/wallets/src/public/images/accounts/passport-placeholder.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="128" height="78" viewBox="0 0 128 78" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
how about minifying the icons?

Reply:
The SVGs are already minified using SVGO when imported through `tsx` files: 
<img width="503" alt="Screenshot 2023-11-14 at 4 01 12 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/5ea2dcdc-e16c-49ae-b199-7f0e99ea0948">